warehouse/models/docs/_docs_payments_views.md

```diff
@@ -0,0 +1,41 @@
+{% docs customer_funding_source_customer_id %}
```

Comment:
are these docs actually referenced in YAML? they won't show up anywhere without that

Reply:
yep - I added them to the YAML, and it looks like they're persisting into the docs, unless I'm missing something
<img width="643" alt="Screen Shot 2022-07-26 at 12 05 55 PM" src="https://user-images.githubusercontent.com/25835059/181055812-469b525e-58a9-42c2-83c5-aa323d8d60be.png">